engine/language-server/src/main/scala/org/enso/languageserver/search/SuggestionsHandler.scala

```diff
@@ -113,7 +113,7 @@ final class SuggestionsHandler(
 
     config.contentRoots.foreach { case (_, contentRoot) =>
       PackageManager.Default
-        .fromDirectory(contentRoot)
+        .fromDirectory(contentRoot.file)
```

Comment:
Now, when we have multiple content roots, `SuggstionsHandler` should take a single content root of type project and read the project definition from it.